cmd/main.go

```diff
@@ -951,3 +982,37 @@ func stopOpenshiftReleaseSyncerIfNeeded(openShiftReleaseSyncer *thread.Thread) {
 		openShiftReleaseSyncer.Stop()
 	}
 }
+
+func addReleaseImagesToDBIfNeeded(db *gorm.DB, releaseImages models.ReleaseImages, failOnError func(err error, msg string, args ...interface{})) {
+	if Options.EnableKubeAPI {
```

Comment:
I think we also need to check release sources here.
Because we want to just let the syncer thread add these if it's going to run rather than adding them here and then having the sync thread immediately delete and readd them.